contracts/GnosisSafe.sol

```diff
@@ -317,16 +314,16 @@ contract GnosisSafe
     * @param _signature Signature byte array associated with _data
     * @return a bool upon valid or invalid signature with corresponding _data
     */
-    function isValidSignature(bytes calldata _data, bytes calldata _signature)
-        external
+    function isValidSignature(bytes memory _data, bytes memory _signature)
+        public
```

Comment:
`public` adds additional processing before the block in order to have JUMPDESTs and a preamble that will arrange CALLDATA into the stack properly. Is the intention to extend the `GnosisSafe` contract itself? I don't see any places where the `internal` part of the `public` modifier gets used.

Reply:
Is that worth it not being compliant with the interface specified at 1271? It's not being used right now, but it might be used in future as I will PR in the next days another improvement to the Gnosis safe

Reply:
Since this non-compliance has 0 functional impact, it has a higher priority for me to touch as little code as possible.